src/transformers/models/roberta_prelayernorm/modeling_roberta_prelayernorm.py

```diff
@@ -963,14 +963,10 @@ def forward(
         Example:
 
         ```python
-        >>> from transformers import (
-        ...     RobertaPreLayerNormTokenizer,
-        ...     RobertaPreLayerNormForCausalLM,
-        ...     RobertaPreLayerNormConfig,
-        ... )
+        >>> from transformers import RobertaTokenizer, RobertaPreLayerNormForCausalLM, RobertaPreLayerNormConfig
```

Comment:
Since fixing the docstring, let's bring it to the new standard: use `AutoTokenizer` and `AutoConfig` here as we are only documenting `RobertaPreLayerNormForCausalLM`.